a-propos/derniers-changements.md

```diff
@@ -11,7 +11,19 @@ title: "Derniers changements apportés au système de conception de Canada.ca"
   <button type="button" class="btn btn-default wb-toggle" data-toggle="{&quot;selector&quot;: &quot;details&quot;, &quot;parent&quot;: &quot;#expand-collapse&quot;, &quot;type&quot;: &quot;off&quot;}">Réduire tout</button>
 </div>
 <div id="expand-collapse">
- <details open="open">
+  <details open="open">
+    <summary>25 septembre 2024</summary>
+    <p>Stabilisation du modèle de page de sujet.</p>
+<p>Dans le cadre de cette mise à jour, nous avons apporté des modifications à plusieurs modèles.</p>
+<p>Changements</p>
+    <ul> 
+<li><a href="{{ site.url }}/configurations-conception-communes/bloc-introduction.html">Bloc d’introduction</a> : ajout d’une option d’image demi-largeur</li>
+<li><a href="{{ site.url }}/configurations-conception-communes/services-renseignements.html">Services et renseignements</a> : ajout de la possibilité de rendre invisible l’en-tête « Services et renseignements »</li>
+<li><a href="{{ site.url }}/configurations-conception-communes/bloc-medias-sociaux.html">Réseaux sociaux</a> : ajout d’une option permettant d’inclure un lien « D’autres moyens de rester en contact »</li>
+<li><a href="{{ site.url }}/common-design-patterns/contributors.html">Contributeurs</a> : changement du design, lequel remplace la <a href="{{ site.url }}/common-design-patterns/institutional-byline.html">configuration d’institution responsable</a></li>
```

Comment:
Both links in this line are giving me errors when I click on them. It looks like they've been coded to point to the English site instead of the French.